Examples/torch/utils/image_net_trainer.py

```diff
@@ -71,7 +71,8 @@ def __init__(self, images_dir: str, image_size: int, batch_size: int = 128,
                                                 is_training=True, num_workers=num_workers,
                                                 num_samples_per_class=num_train_samples_per_class).data_loader
 
-        self._evaluator = ImageNetEvaluator(images_dir=images_dir, image_size=image_size, batch_size=batch_size)
+        self._evaluator = ImageNetEvaluator(images_dir=images_dir, image_size=image_size, batch_size=batch_size, 
+                                            num_workers=num_workers)
```

Comment:
add num_workers to prevent always using the default number of num_workers (32)

Reply:
Hi @shiannn - the pylint checker is very sensitive :) Its complaining that you have trailing whitespace on this line. Please remove that and push again. Thanks.

Reply:
Hi @quic-akhobare - Fixed. Thanks!